### PR TITLE
Updated ThingsBoard Arduino SDK link

### PR DESCRIPTION
**Pull request by imbeacon**
Hi, the link of the repository was changed from https://github.com/thingsboard/thingsboard-arduino-mqtt-sdk to https://github.com/thingsboard/thingsboard-arduino-sdk

### CODE REVIEW

**Comment by imbeacon** [comment]
@ArduinoBot Uplated.

**Comment by imbeacon** [comment]
@ArduinoBot Created new release.

**Review by per1234 — COMMENTED** [review]
Hello @imbeacon. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/2443#issuecomment-1413765208

**Pull request merged**
